generate/template/astronomy.kt

```diff
@@ -473,13 +473,15 @@ class Time private constructor(
     internal fun julianMillennia() = tt / DAYS_PER_MILLENNIUM
 
     companion object {
+        @JvmStatic
         private val origin = GregorianCalendar(TimeZoneUtc).also {
```

Comment:
probably only public fields and methods should be marked with static but well maybe it instructs the compiler to optimize companion object altogether or not, not sure.

Reply:
I seem to remember we talked about whether we want to get rid of using `GregorianCalendar`, because it brings in a dependency on Java. Is that correct? I think you said if someone wants to use the Kotlin version of Astronomy Engine to create WebAssembly, this could be a problem. I think there a few other minor uses of Java symbols. It is possible to do something like the C version, where we make our own date/time functions from scratch.

Reply:
Oh sure, you can just take unix timestamp, the core data java.util.Date holds, https://docs.oracle.com/javase/7/docs/api/java/util/Date.html#getTime() this is a roughly separate matter from what this change is about, https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.jvm/-jvm-inline/ is considered to be common between Kotlin targets mainly for Java support for sure.

Reply:
OK, I think you meant `@JvmStatic`, not `@JvmInline`, but it looks like `@JvmStatic` is also safe to use outside the JVM (if I understand the documentation correctly). I just keep thinking about this, and yes, I understand it is a separate issue. I just wanted to understand it. Thank you!